src/Endpoints/WorkersKV.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * User: junade
+ * Date: 01/02/2017
+ * Time: 12:30
+ */
+
+namespace Cloudflare\API\Endpoints;
+
+use Cloudflare\API\Adapter\Adapter;
+use Cloudflare\API\Traits\BodyAccessorTrait;
+
+class WorkersKV implements API
+{
+    use BodyAccessorTrait;
+
+    private $adapter;
+
+    public function __construct(Adapter $adapter)
+    {
+        $this->adapter = $adapter;
+    }
+
+    public function createNamespace(string $accountID, string $namespace): \stdClass
+    {
+        $response = $this->adapter->post(
+            'accounts/' . $accountID . '/storage/kv/namespaces',
+            ['title' => $namespace]
+        );
+        $this->body = json_decode($response->getBody());
+        return $this->body->result;
+    }
+
+    public function getNameSpaces(string $accountID): array
+    {
+        $response = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces');
+        $this->body = json_decode($response->getBody());
+        return $this->body->result;
+    }
+
+    public function listNamespaceKeys(string $accountID, $namespaceIdentifier): array
+    {
+        $response = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/keys');
+        $this->body = json_decode($response->getBody());
+        return $this->body->result;
+    }
+
+    public function getReadKeyValuePair(string $accountID, $namespaceIdentifier, $keyName): string
+    {
+        $response = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/values/' . $keyName);
+        $this->body = json_decode($response->getBody());
```

Comment:
My understanding is that the `Content-Type` for these responses is `application/octet-stream`, so we shouldn't be `json_decoding` here. Since the type returned by `$response->getBody()`, `StreamInterface`, supports `__toString()` and seems to do the right thing, it might be worth changing this to:
```diff
-        $this->body = json_decode($response->getBody());
+        $this->body = (string) $response->getBody();
```